scripts/zones/Port_Jeuno/npcs/Shami.lua

```diff
@@ -87,38 +87,89 @@ function onTrade(player,npc,trade)
 ----------------------------------------------------------------------------------------------------------------------------
 -------- Trading Cracked KSNM Orbs or checking where you can bring the Orb -------------------------------------------------
 ----------------------------------------------------------------------------------------------------------------------------
-        
-    elseif (trade:hasItemQty(1180,1) and NumberItem == 1) then  
+
+    elseif (trade:hasItemQty(1180,1) and NumberItem == 1) then
         if (player:getVar("AtroposOrbIsCracked") == 1) then
             player:startEvent(0x0016); -- Atropos Orb is Cracked
             player:setVar("AtroposOrbIsCracked",0);
         else
             player:startEvent(0x0009); -- Atropos Orb is ok, List where you can take the orb.
         end
-    elseif (trade:hasItemQty(1175,1) and NumberItem == 1) then  
+    elseif (trade:hasItemQty(1175,1) and NumberItem == 1) then
         if (player:getVar("ClothoOrbIsCracked") == 1) then
             player:startEvent(0x0016); -- Clotho Orb is Cracked
             player:setVar("ClothoOrbIsCracked",0);
         else
             player:startEvent(0x0009); -- Clotho Orb is ok, List where you can take the orb.
-        end        
-    elseif (trade:hasItemQty(1178,1) and NumberItem == 1) then  
+        end
+    elseif (trade:hasItemQty(1178,1) and NumberItem == 1) then
         if (player:getVar("LachesisOrbIsCracked") == 1) then
             player:startEvent(0x0016) -- Lachesis Orb is Cracked
             player:setVar("LachesisOrbIsCracked",0);
         else
             player:startEvent(0x0009); -- Lachesis Orb is ok, List where you can take the orb.
         end
-    elseif (trade:hasItemQty(1553,1) and NumberItem == 1) then  
+    elseif (trade:hasItemQty(1553,1) and NumberItem == 1) then
         if (player:getVar("ThemisOrbIsCracked") == 1) then
             player:startEvent(0x0016); -- Themis Orb is Cracked
             player:setVar("ThemisOrbIsCracked",0);
         else
             player:startEvent(0x000b); -- Themis Orb is ok, List where you can take the orb.
         end
--- TODO : Add in trade-in for cracked Kindred Crest orbs (and High Kindred Crest) and find cutscene that tells you where you can bring it        
+----------------------------------------------------------------------------------------------------------------------------
+-------- Trading Cracked KCNM Orbs or checking where you can bring the Orb -------------------------------------------------
+----------------------------------------------------------------------------------------------------------------------------
+    elseif (trade:hasItemQty(3351,1) and NumberItem == 1) then
+        if (player:getVar("PhobosOrbIsCracked") == 1) then
+            player:startEvent(0x0016); -- Phobos Orb is Cracked
+            player:setVar("PhobosOrbIsCracked",0);
```

Comment:
that was the old line, just moved it to add the missing content (whoever was supposed to add)